### PR TITLE
In old release versions run small package of most important tests

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because we want to run only the most important test on old release versions

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://marketplace-gray.vercel.app/

### Do you want to run more stable tests?
Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation

CONTAINERS=1

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-8149 For older version of saleor run only critical tests](https://app.clickup.com/t/2549495/SALEOR-8149)

**Comment by karola312** [comment]
@eCzerniak please check if the correct tests have oldRelease tag

**Comment by karola312** [comment]
@eCzerniak Added

**Review by eCzerniak — DISMISSED** [review]

File: .github/workflows/tests-nightly.yml [hunk 1]
```diff
@@ -201,15 +203,28 @@ jobs:
...
+        env:
+          is_old_version: ${{ needs.get-environment-variables.outputs.is_old_version }}
+          old_version_test_tag: "@oldRelease"
+          new_version_test_tag: "@stable"
+        run: |
+          if [[ $is_old_version == "true" ]]; then
+              echo "::set-output name=TEST_TAG::$old_version_test_tag "
+          else
+              echo "::set-output name=TEST_TAG::$new_version_test_tag "
+          fi
```
Comment (NyanKiyoshi) [inline .github/workflows/tests-nightly.yml:218]: Deprecated

File: .github/workflows/tests-nightly.yml [hunk 2]
```diff
@@ -219,13 +234,14 @@ jobs:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           CYPRESS_mailHogUrl: ${{ secrets.CYPRESS_MAILHOG }}
-          COMMIT_INFO_MESSAGE: Critical triggered via release - ${{github.event.client_payload.project}} ${{github.event.client_payload.version}}, ${{github.event.client_payload.pullRequestUrl}}
-          CYPRESS_grepTags: '@stable'
+          COMMIT_INFO_MESSAGE: Triggered via release - ${{github.event.client_payload.project}} ${{github.event.client_payload.version}}, ${{github.event.client_payload.pullRequestUrl}}
+          CYPRESS_grepTags: '${{steps.set-tag-for-tests.outputs.TEST_TAG}}'
```
Comment (NyanKiyoshi) [inline .github/workflows/tests-nightly.yml:238]:
```suggestion
          CYPRESS_grepTags: ${{steps.set-tag-for-tests.outputs.TEST_TAG}}
```

**Review by NyanKiyoshi — DISMISSED** [review]

**Review by NyanKiyoshi — APPROVED** [review]

**Review by szczecha — APPROVED** [review]

**Pull request merged**